libkbfs/folder_block_ops.go

```diff
@@ -308,11 +308,11 @@ func (fbo *folderBlockOps) getCleanEncodedBlockSizeLocked(ctx context.Context,
 		return 0, InvalidBlockRefError{ptr.Ref()}
 	}
 
-	// Try to get the encoded size from the cache before escalating to the
-	// block retriever (even though it's supposed to do a similar thing with
-	// checking caches before checking the data version).
-	// TODO: Figure out how to remove this without breaking journal tests in
-	// kbfs/test.
+	// Try to get the encoded size from the cache before escalating to
```

Comment:
Are you using 8-space tabs? This shows up as <80chars with 4-space tabs in my editor.

Reply:
Nope, 4-space.  Everything looks right to me, in my editor and in github.

Reply:
What's your column limit?

Reply:
80.
![80chars](https://user-images.githubusercontent.com/8516691/30403090-f5f0982e-9894-11e7-84a8-b19aa328cb02.png)

Reply:
Oh, you're complaining because it's less than 80 chars?  I just used alt-q in emacs, like I always do.  It doesn't push things right to the edge.

Reply:
I'm just trying to get us to a place where when I reflow something in my client, I know that your client won't do it, because often it's distracting in PRs to have reflowed text mixed with actual code changes.

Reply:
Ok sorry.  I just use: http://ergoemacs.org/emacs/modernization_fill-paragraph.html, which says "~70 characters".  That's what I've done since the beginning of KBFS.  Not sure how to translate that to other editors.